milestone2.ts

```diff
@@ -164,7 +189,15 @@ let getVolunteerInfo: GetVolunteerInfo = (vol) => {
 /* these criteria, return undefined
  */
 
-let daBigTest: GetVolunteer; // Code here
+type DaBigTest = (data: Volunteer[]) => Volunteer | undefined;
+let daBigTest: GetVolunteer = (data) => {
+  let fromSlo = data.find(x => x.city === "SLO" && x.age > 40);
+  if (fromSlo === undefined)
+    return undefined;
+  let firstClone = {...fromSlo};
+  firstClone.position = "staff";
```

Comment:
you could have combined these steps by spreading the updated field into the clone when you were making it, similar to how you did it in question 6. This works but isn't as concise/readable